src/tzstats/tzstats_reward_provider_helper.py

```diff
@@ -30,6 +30,8 @@ def __init__(self, nw, baking_address):
 
     def get_rewards_for_cycle(self, cycle, expected_reward = False, verbose=False):
         #############
+        ## TODO: Fetch current balance to check if zeroed
```

Comment:
The implementation for tzstats is still pending. Code usability for tzstats should be ensured before the code could be merged.

Reply:
Going to need some help here as I've never used tzstats for payouts. Will require research into tzstats API to get needed information.

Reply:
Nah, nevermind. tzstats API actually makes this pretty easy. I'll have this fixed later tonight possibly.